test-integration/test_integration/test_redis_queue.py

```diff
@@ -51,6 +52,7 @@ def test_queue_worker_files(docker_image, docker_network, redis_client, upload_s
         )
         response = json.loads(redis_client.brpop("response-queue", timeout=10)[1])
         assert response == {
+            "timestamp": mock.ANY,
```

Comment:
It looks like all the assertions are using `mock.ANY`. Do you think it's worth having at least one test that asserts the value is indeed an ISO formatted date string?